plugin/src/main/java/com/playmonumenta/scriptedquests/zones/event/ZoneEvent.java

```diff
@@ -0,0 +1,34 @@
+package com.playmonumenta.scriptedquests.zones.event;
+
+import com.playmonumenta.scriptedquests.utils.NmsUtils;
+import org.bukkit.Bukkit;
+import org.bukkit.block.Block;
+import org.bukkit.entity.Entity;
+import org.bukkit.entity.Player;
+
+public abstract class ZoneEvent {
```

Comment:
There's already two zone events with more descriptive names; this sounds like it's a base class for those and should be renamed.

Reply:
Named correctly, will leave